src/utils/index.js

```diff
@@ -74,3 +74,26 @@ export const setUrlParams = (section) => {
     window.history.replaceState(null, null, params);
   }
 };
+
+export const getCategoryTitle = (menuItems) => {
+  const pathname = window.location.pathname;
+
+  var categoryTitle = "";
+
+  const iterate = (obj) => {
+    if (obj.url === pathname) {
+      categoryTitle = obj.title;
+    }
+    if (obj.items && obj.items.length > 0) {
```

Comment:
maybe just `obj.items?.length > 0`

Reply:
It causes some webpack issues. Support for the experimental syntax 'optionalChaining' isn't currently enabled there